doc/source/whatsnew/v1.0.0.rst

```diff
@@ -52,7 +52,7 @@ Other API changes
 
 Deprecations
 ~~~~~~~~~~~~
-
+- :func:`pandas.json_normalize` is now exposed in the top-level namespace. Usage of :func:`json_normalize` as :func:`pandas.io.json.json_normalize` is now deprecated and it is recommended to use :func:`json_normalize` as :func:`pandas.json_normalize` instead. (:issue:`27586`)
```

Comment:
I would remove the :func:\`pandas.io.json.json_normalize\` as it's the deprecated one. Just use double-backticks instead.

Reply:
Sure, thank you for the pointer.